test/puppetlabs/puppetdb/http/nodes_test.clj

```diff
@@ -69,57 +68,61 @@
 ;; TESTS
 
 (deftestseq node-queries
-  [[version endpoint] endpoints]
+  [[version endpoint] endpoints
+   method [:get :post]]
 
-  (let [{:keys [web1 web2 db puppet]} (store-example-nodes)]
+  (let [status-for-node' #(status-for-node method endpoint %)
```

Comment:
Where do we stand on point-free-style? I know AJ likes it, you appear to mostly avoid it, and I'm wishy washy as usual.